app/src/main/java/com/github/ybecker/epforuml/chat/ChatHomeAdapter.kt

```diff
@@ -34,7 +35,16 @@ class ChatHomeAdapter(private val chatList : MutableList<String>,private val mai
          DatabaseManager.db.getUserById(currentItem).thenAccept{
              val user : Model.User? = it
              holder.chatWithText.text = user?.username
-             if(user?.profilePic!="") holder.chatImage.setImageURI(Uri.parse(user?.profilePic))
+
+             // Load profile picture to image view
+             if (user != null) {
+                 AccountFragment.loadProfilePictureToView(

```

Comment:
This code seems to be repeated many times here and in ChatAdapter.kt. Maybe you could have created a method to execute it.

Reply:
I don't think it's worth as it's just a call to a method (with sometimes a null check on the user)